### PR TITLE
Mkdocs src

### PR DESCRIPTION
**Pull request by alitariq4589**
- I have added mkdocs_src folder containing all the `.md` files. Also this branch is sourced from `gh-branch`.  
- Removed `mkdocs/site` folder and `mkdocs.yml` file
- Removed `mkdocs_src/.vscode`.  
- Added `Installing_ssl_certificates_new.md` which contains documentation for configuring letsencrypt with jenkins.
- Changed documentation title from `RISC-V CI` docs to `Cloud-V docs`  
- Added `.md` file for cross-compiling rust
- Added `.md` file for cross-compiling python3.8.15
- Added `.md` file for cross-compiling go

### CODE REVIEW

File: mkdocs_src/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mmhus) [inline mkdocs_src/.vscode/settings.json:1]: Remove file.

File: mkdocs_src/site/404.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```
Comment (mmhus) [inline mkdocs_src/site/404.html:1]: `mkdocs_src/site` dir shouldn't be included.

**Review by mmhus — CHANGES_REQUESTED** [review]
Pls update with changes.

File: mkdocs_src/site/404.html (hunk 2 above)
Comment (alitariq4589) [reply mkdocs_src/site/404.html]: I have removed file and folder and updated the description

File: mkdocs_src/.vscode/settings.json (hunk 1 above)
Comment (mmhus) [reply mkdocs_src/.vscode/settings.json:1]: @alitariq4589 You didn't remove this

**Review by mmhus — CHANGES_REQUESTED** [review]
Make a habit to review carefully before adding the changes, and resolve all comments before asking for review.

File: mkdocs_src/site/404.html (hunk 2 above)
Comment (mmhus) [reply mkdocs_src/site/404.html]: Resolve conversations you have fixed.

File: mkdocs_src/.vscode/settings.json (hunk 1 above)
Comment (alitariq4589) [reply mkdocs_src/.vscode/settings.json]: I removed `mkdocs_src/.vscode` file now. Actually it was hidden so I could not see it last time.

**Review by mmhus — APPROVED** [review]

**Pull request merged**